CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Minor: Added `/shoutout <username>` commands to shoutout specified user. (#4638)
 - Minor: Improved editing hotkeys. (#4628)
 - Bugfix: Fixed generation of crashdumps by the browser-extension process when the browser was closed. (#4667)
+- Bugfix: Fixed input-completion popup and quick-switcher not respecting the current theme. (#4671)
```

Comment:
Change this to a `Minor` entry with the tone of "the input completion & quick switcher are now styled to match your theme" or something